resources/less/virkailija-application.less

```diff
@@ -133,6 +144,23 @@
   font-size: 13px;
   line-height: 21px;
 }
+.application-handling__popup__mass-notes {
```

Comment:
Ei liity tähän kyseiseen riviin, mutta Lessissä voisi groupata näitä tyylejä sisäkkäin jolloin on helpompi lukea ja hanskata sisäkkäisiä tyylejä, mahdollisesti myös pääsee vähemmillä luokilla sillä tavalla

Reply:
joo mietin itsekin että nää voisi tehdä fiksummin